packages/tx/examples/EOACodeTx.ts

```diff
@@ -1,22 +1,26 @@
 import { Common, Hardfork, Mainnet } from '@ethereumjs/common'
-import { createEOACode7702Tx } from '@ethereumjs/tx'
-import { type PrefixedHexString, createAddressFromPrivateKey, randomBytes } from '@ethereumjs/util'
+import { Capability, createEOACode7702Tx } from '@ethereumjs/tx'
+import {
+  type EOACode7702AuthorizationListItem,
+  type PrefixedHexString,
+  createAddressFromPrivateKey,
+  randomBytes,
+} from '@ethereumjs/util'
 
-const ones32 = `0x${'01'.repeat(32)}` as PrefixedHexString
+const ones32: PrefixedHexString = `0x${'01'.repeat(32)}`
+const authorizationListItem: EOACode7702AuthorizationListItem = {
+  chainId: '0x2',
+  address: `0x${'20'.repeat(20)}`,
+  nonce: '0x1',
+  yParity: '0x1',
+  r: ones32,
+  s: ones32,
```

Comment:
This does not show how to sign the authorization list item, which is necessary, because the authorization signer does not necessarily have to be the tx signer.

Reply:
I agree, I just refactored this one initially (didn't actually work on it) so that it had at least correct types. We can certainly improve it.